source/globalCommands.py

```diff
@@ -459,17 +459,30 @@ def script_toggleReportLineNumber(self,gesture):
 	script_toggleReportLineNumber.category=SCRCAT_DOCUMENTFORMATTING
 
 	def script_toggleReportLineIndentation(self,gesture):
-		if config.conf["documentFormatting"]["reportLineIndentation"]:
+		lineIndentationSpeech = config.conf["documentFormatting"]["reportLineIndentation"]
+		lineIndentationTones = config.conf["documentFormatting"]["reportLineIndentationWithTones"]
+		if not lineIndentationSpeech and not lineIndentationTones:
 			# Translators: The message announced when toggling the report line indentation document formatting setting.

```

Comment:
This is now one of several possible messages, so this should say:

`A message reported when cycling the report line indentation document formatting setting.`